### PR TITLE
update jhipster to 7.9.3

### PR DESCRIPTION
**Pull request by atomfrede**
Update jhipster version to 7.9.3

### CODE REVIEW

**Comment by mshima** [comment]
Failures looks related to https://github.com/jhipster/generator-jhipster/pull/19301.
Windows failures are related to npmw.cmd changes.

**Comment by atomfrede** [comment]
Will have a look

**Comment by mshima** [comment]
At windows, we should run with `—skip-install` and run `npm install later`, I can take a look later.

**Comment by atomfrede** [comment]
I will try to have a look at the oauth part this evening. JWT seems to work fine indeed.

**Comment by atomfrede** [comment]
It made it work with caffeine. Right now this will only support caffeine directly for the oauth caching. Spring cache integration seems not to work, so our recommendation, to run this without a cache provider still stands imho.

**Comment by atomfrede** [comment]
> At windows, we should run with `—skip-install` and run `npm install later`, I can take a look later.

@mshima Can you give me hint where to look at? It is something we should configure in the ci pipeline or in generated code?

**Comment by mshima** [comment]
@atomfrede sure.
It's related to https://github.com/jhipster/generator-jhipster/pull/19404.
GraalVM adds a lot of paths to `$PATH`, https://github.com/jhipster/generator-jhipster/pull/19404 added more path overflowing the max size of `$PATH`.
The workaround is to execute `jhipster-native jdl` command with `--skip-install` and run `npm install` later.

**Comment by mshima** [comment]
I will do the initial change.

**Comment by atomfrede** [comment]
Thanks. I remember that with bootrun and gradle some time ago

**Comment by atomfrede** [comment]
Looks good. Failing macos tests fail to install docker in the pipeline, so I would say good to merge imho. Thanks @mshima for doing the windows ci part!

File: .github/workflows/jdl.yml [hunk 1]
```diff
@@ -172,13 +174,19 @@ jobs:
       # Install JHipster and generate project+entities
       #----------------------------------------------------------------------
       - name: 'PROJECT: generate'
-        run: 'JHI_PROFILE=${{ matrix.profile }} jhipster-native jdl "${{ github.workspace }}/generator-jhipster-native/test-integration/samples/${{ matrix.suite }}.jdl" --build ${{ matrix.build-tool }} --no-insight --skip-git --skip-commit-hook --skip-jhipster-dependencies --no-cypress-audit'
+        run: 'JHI_PROFILE=${{ matrix.profile }} jhipster-native jdl "${{ github.workspace }}/generator-jhipster-native/test-integration/samples/${{ matrix.suite }}.jdl" --build ${{ matrix.build-tool }} --no-insight --skip-git --skip-commit-hook --skip-jhipster-dependencies --no-cypress-audit ${{ matrix.jdl-extra-args }}'
+
+      - name: 'Windows: force npm to use bash'
```
Comment (mshima) [inline .github/workflows/jdl.yml:179]:
```suggestion
      - name: 'Windows: install npm dependencies'
```

**Comment by mshima** [comment]
Thanks @atomfrede

**Pull request merged**